### DISCUSSION TITLE
Use Combobox as a custom Time/Datepicker Component

### DISCUSSION THREAD

**Discussion by timheerwagen** [tailwindlabs/headlessui#2380] [Help]
First of all, thanks for the great library, I've been a big fan for a long time.

My question is if it is possible or recommended to use a combobox as a custom time/datepicker?

I have tried to give Combobox.Input type="time", but this errors directly at the input.

I don't need any options like with autocomplete, but it would be sufficient to have a popover appear where selections can be made, but not necessarily lose focus from the input. The rest of the logic i can handle by myself inside this container.

However, I have already tried this with a popover, but then I can no longer make an entry in the input when I click.

Thank you very much for your help!

**Comment by D3leVv** [comment]
Hi, may i ask why do you need to use Combobox if you want to render the native calendar ??? Just use regular input with the type="time" or "date"

**Reply by timheerwagen** [reply]
Because you cannot style the native time input and add additional behaviour.

Also, this was just an example, you could do much more than a time input with a combo box.

**Reply by D3leVv** [reply]
Thats great, but that still doesn’t answer my question. If you don’t want to use the native input styles, why are you trying to use it inside the custom component?

**Reply by timheerwagen** [reply]
I'm sorry if I misunderstood the question. I tried Combobox.Input type="time",
because I wanted to see if I could keep the native masking of the input.

**Reply by D3leVv** [reply]
Mhm i don’t think this is possible with that component
